kinesis/src/main/scala/com/amazonaws/services/kinesis/scala/AmazonKinesisClient.scala

```diff
@@ -37,34 +37,22 @@ class AmazonKinesisClient(
 
   def this(region: Regions) = this(Region.getRegion(region))
 
-  def describeStream(request: com.amazonaws.services.kinesis.model.DescribeStreamRequest):
-      Future[com.amazonaws.services.kinesis.model.DescribeStreamResult] = {
-
-    invoke(client.describeStreamAsync, request)
-  }
-
-  def getShardIterator(request: com.amazonaws.services.kinesis.model.GetShardIteratorRequest):
-      Future[com.amazonaws.services.kinesis.model.GetShardIteratorResult] = {
-
-    invoke(client.getShardIteratorAsync, request)
-  }
-
-  def listTagsForStream(request: com.amazonaws.services.kinesis.model.ListTagsForStreamRequest):
-      Future[com.amazonaws.services.kinesis.model.ListTagsForStreamResult] = {
+  def addTagsToStream(request: com.amazonaws.services.kinesis.model.AddTagsToStreamRequest):
```

Comment:
Reminder to self: would be nice to deterministically sort these so things don't get reordered like this when we regenerate.